docs/reference/elasticsearch/rest-apis/retrievers/diversify-retriever.md

```diff
@@ -0,0 +1,85 @@
+---
+applies_to:
+  stack: all
+  serverless: ga
```

Comment:
```suggestion
  stack: ga 9.3
  serverless: ga
```

needs a lifecycle and a version number

is this retriever straight to GA? 

if not `s/ga/preview` here